### PR TITLE
Push notification for query response

### PR DESCRIPTION
**Pull request by Subbu-betalectic**

### CODE REVIEW

File: src/functions/neptune/neptuneCaller.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  var data = {
+    version: 1,
+    message_id: process.env.NEPTUNE_ENV + "-" + uuid.v4(),
+    timestamp: dayjs().unix(),
+  };
+  data = JSON.stringify({ ...data, ...payload });
+  console.log("chekc here data:::>", data);
+  var config = {
+    method: "post",
+    url: `${process.env.NEPTUNE_ENDPOINT}/${process.env.NEPTUNE_ENV}/events`,
```
Comment (rajeshn95) [inline src/functions/neptune/neptuneCaller.js]: @Subbu-betalectic Can't we use neptune sdk here? https://www.npmjs.com/package/@teurons/neptune-nodejs

File: src/stories/Responses/UserCanAddResponse/story.js [hunk 2]
```diff
@@ -58,7 +64,27 @@ const validateInput = async (prepareResult) => {
 const handle = async ({ prepareResult, authorizeResult }) => {
   try {
     await validateInput(prepareResult);
-    return ResponseRepo.create(prepareResult);
+    let res = await ResponseRepo.create(prepareResult);
+    console.log("res>>>>>", res, JSON.stringify(res));
```
Comment (rajeshn95) [inline src/stories/Responses/UserCanAddResponse/story.js]: @Subbu-betalectic Remove the console logs

Comment (Subbu-betalectic) [reply src/stories/Responses/UserCanAddResponse/story.js]: Removed the console logs

File: src/functions/neptune/neptuneCaller.js (hunk 1 above)
Comment (Subbu-betalectic) [reply src/functions/neptune/neptuneCaller.js]: Now using the neptune  sdk

**Review by rajeshn95 — APPROVED** [review]

**Pull request merged**